pareto/other_models/CM_module/operational/screening_tool.ipynb

```diff
@@ -0,0 +1,801 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Assessing lithium revenue possibilities\n",
+    "\n",
+    "Now we will try to assess whether the current infrastructure is hindering lithium revenue opportunities. \n",
+    "\n",
+    "In the next block we will rerun the model, but this time to maximize lithium revenue without considering the impact that has on other operating costs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "   ... running linear flow model\n",
+      "   ... running bilinear model\n",
+      "Max lithium revenue with existing infrastructure: 870608.1283902546\n",
+      "Arc cost:                              71795\n",
+      "Disposal cost:                       1117956\n",
+      "Freshwater cost:                     7751282\n",
+      "Treatment cost:                      2255071\n",
+      "Storage cost:                         314202\n",
+      "Storage revenue:                      214870\n",
+      "Lithium revenue:                      870608\n",
+      "Beneficial Reuse Revenue:             696487\n",
+      "Net cost:                            9728341\n"
+     ]
+    }
+   ],
+   "source": [
+    "max_inf_model = max_recovery_with_infrastructure(data)\n",
+    "print_results_summary(max_inf_model)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As we can see the maximum Lithium revenue that can be attained given the existing infrastructure is $8,70,608/yr. \n",
+    "\n",
+    "Now we will compare this number to the maximum theoretical Li revenue that can be obtained from this network if we disregard the existing infrastructure."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Max. lithium revenue with\n",
+      " existing infrastructure:    870608\n",
+      "\n",
+      "Max theoretical treatment revenue\n",
+      " (ignoring infrastructure):  1173483\n"
+     ]
+    }
+   ],
+   "source": [
+    "max_recovery = max_theoretical_recovery_flow_opt(\n",
+    "    model,\n",
+    "    treatment_unit=\"R01_IN\",  # change R01 if you want to consider a different treatment unit\n",
+    "    desired_li_conc=100,\n",
+    ")\n",
+    "\n",
+    "# Displaying difference between max possible lithium revenue with and without existing infrastructure\n",
+    "print(\n",
+    "    f\"Max. lithium revenue with\\n existing infrastructure:  \"\n",
+    "    f\"{pyo.value(max_inf_model.treat_rev):>8.0f}\"\n",
+    ")\n",
+    "print()\n",
+    "print(\n",
+    "    f\"Max theoretical treatment revenue\\n (ignoring infrastructure): \"\n",
```

Comment:
```suggestion
    "    f\"Max. theoretical treatment revenue\\n (ignoring infrastructure): \"\n",
```
Be consistent as to whether you're using "max" or "max." (it doesn't matter which option you prefer but please be consistent).